system_files/usr/share/ublue-os/just/50-devmode.just

```diff
@@ -0,0 +1,50 @@
+
+alias devmode := change-edition
+# Allows selection of Bluefin LTS editions
+change-edition:
+    #!/bin/bash
+
+    # Function to display the menu and get user selection using gum
+    select_edition() {
+    edition=$(gum choose --header "Please select a edition:" "HWE" "GDX" "DX" "Base")
+
+    if [[ -z "$edition" ]]; then # Handle cancelation
+        echo "No Edition selected."
+        exit 1
+    fi
+
+    echo "You have selected edition: $edition"
+
+    # Consider Base as empty string
+    if [[ "$edition" == "Base" ]]; then
```

Comment:
We don't have a concept of "editions", IMO just match the text that bluefin uses.